Shared/Samples/Run valve isolation trace/RunValveIsolationTraceView.swift

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct RunValveIsolationTraceView: View {
+    /// The view model for the sample.
+    @StateObject var model = Model()
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(
+                map: model.map,
+                graphicsOverlays: [model.parametersOverlay]
+            )
+            .onSingleTapGesture { screenPoint, mapPoint in
+                model.lastSingleTap = (screenPoint, mapPoint)
+            }
+            .overlay(alignment: .top) {
+                Text(model.statusText)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.ultraThinMaterial, ignoresSafeAreaEdges: .horizontal)
+                    .multilineTextAlignment(.center)
+            }
+            .task {
+                try? await model.setup()
+                await mapViewProxy.setViewpointCenter(model.startingLocationPoint, scale: 3_000)
+            }
+            .task(id: model.lastSingleTap?.mapPoint) {
+                guard let lastSingleTap = model.lastSingleTap else {
+                    return
+                }
+                if let feature = try? await mapViewProxy.identifyLayers(
+                    screenPoint: lastSingleTap.screenPoint,
+                    tolerance: 10
+                ).first?.geoElements.first as? ArcGISFeature {
+                    model.addFilterBarrier(for: feature, at: lastSingleTap.mapPoint)
+                }
+            }
+            .toolbar {
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Toggle("Isolated\nFeatures", isOn: $model.includesIsolatedFeatures)
```

Comment:
To me it seems like the button text should have the word "includes".  Something like "Include Isolated Features".

The problem is there isn't a lot of space.  But you could solve that by using a vstack and positioning the status text above the toggle and trace/reset button.

Reply:
I think the text "Isolated\nFeatures" is okay. Here is what the suggested text looks like:

<img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/20174bca-3372-47b8-b55c-6c40c6bd9bf0" width="300">
<img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/815b6bd2-4add-40ae-8fd9-1caad42ec292" width="300">

I'm not sure how to left align the toggle switch here.

Reply:
To me it's too much on one "line".  Can the category button be moved above? Or can the toggle be moved above/below?

Reply:
Does this layout look better?

<img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/1a938472-bdb9-4bbf-a334-bea137e50e5a" width="300">

Reply:
I think something like that is better. Looks like Category button could use some top padding. Also consider putting Category and Reset on the same line. One on right, one on left.

Reply:
Here is the layout with Category and Reset on the same line. Is there a way to increase the height of the toolbar?

<img src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/6d793a5f-6907-4f20-b1b7-27ae9b1fe845" width="300">

Reply:
It looks like you need a `.padding([.top, .bottom])` modifier on the HStack

Reply:
I added padding as suggested, however, the toolbar appears to have a fixed height since the actual padding is less than expected.